aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+
+### Highlights section
+| #  | Section    | Description                                                                                                      | Mandatory/Highly Recommended                                              | Sample Example                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|----|------------|------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| H1 | Highlights | Summarize algorithm performance metric                                                                           | Mandatory                                                                 | In benchmarks from the AutoGluon-Tabular paper, AutoGluon outperformed many popular open-source/commercial AutoML platforms on 50 classification/regression datasets from Kaggle/OpenML. AutoGluon is faster, more robust, and much more accurate than other tools, even outperforming the best-of five other AutoML platforms on most datasets. In two popular Kaggle competitions, AutoGluon beat 99% of the participating data scientists after just 4 hours of training. |
```

Comment:
by convention/style, numbers ten and under should be written out when included in sentences like this:
```suggestion
| H1 | Highlights | Summarize algorithm performance metric                                                                           | Mandatory                                                                 | In benchmarks from the AutoGluon-Tabular paper, AutoGluon outperformed many popular open-source/commercial AutoML platforms on 50 classification/regression datasets from Kaggle/OpenML. AutoGluon is faster, more robust, and much more accurate than other tools, even outperforming the best-of five other AutoML platforms on most datasets. In two popular Kaggle competitions, AutoGluon beat 99% of the participating data scientists after just four hours of training. |
```